specification/dimensions/skuid.md

```diff
@@ -0,0 +1,91 @@
+## Spec Content
+
+### SKUID
+
+A SKU ID is an identifier assigned to the SKU that’s associated with a charge. SKU ID can be referenced on a price list published by a provider to look up detailed information about the SKU including the different unit prices associated with the SKU.
+
+The SKUId column MUST be present in the billing data. This column MUST be of type String. The SKUId MUST NOT be null when [Charge Type](#ChargeType) is ‘Purchase’ or ‘Usage’. The SKUId SHOULD NOT be null when [Charge Type](#ChargeType) is ‘Adjustment’ or [Adjustment Type](#AdjustmentType) is ‘Refund’.
+
+SKUID may have a collection of nullable attributes such as Service, Region, instance type, etc. Future versions of the FOCUS specification may include more precise definitions of those attributes.  In the interim, the SKU ID can be used in concert with an externally-sourced price sheet to facilitate practitioner data modeling and analysis. 
+
+A SKU ID doesn’t necessarily define the price as there may be additional attributes such as tier, region, etc. of the SKU ID that map to a price. 
+
+Every SKU has one or more SKU Prices.  For providers that define their prices at the SKU level, the SKU ID and SKU Price ID shall be the same.
+
+
+#### Column ID
+
+SKUId
+
+
+#### Display name
+
+SKU ID
+
+
+#### Description
+
+A provider-assigned identifier that maps to a unique product offering that has a charge associated with it.
+
+
+#### Content constraints
+
+|         Constraint             | Value                   |
+| -------------------- | ------------------ |
+| Column required | True          |
+|  Data type       |  String        |
+|  Allows nulls    |  True          |
+| Value format    | string-format |
+
+
+#### Introduced (version)
+
+1.0
+
+
+## Supporting content
```

Comment:
This all goes into a separate supporting content file under /supporting_content with the same file name

Reply:
PR #253 created for supporting_content update. Deleted items from file.